config/dev.webpack.config.js

```diff
@@ -1,13 +1,23 @@
-
 const { resolve } = require('path');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
-    https: false,
     port: 8002,
+    useFileHash: false,
```

Comment:
Add `isChrome2: false` for time being in here to use fallback render. Once the code hits production and we update the `main.yml` config to use the module you can remove it from here.

Reply:
Hmm it seems `isChrome2` is part of insights-chrome, and not part of `frontend-components-config`. `skipChrome: true` works though.